ios/FluentUI/Table View/ListVnext.swift

```diff
@@ -0,0 +1,313 @@
+                            }
+                        }
+                    }
+                    Spacer()
+                    HStack {
+                        if let trailingIcon = cell.trailingIcon {
+                            if trailingIcon != .none {
+                                let isDisclosure = trailingIcon == .disclosure
+                                let disclosureSize = tokens.disclosureSize
+                                let iconSize = tokens.iconSize
+                                Image(uiImage: trailingIcon.icon!)
+                                    .resizable()
+                                    .foregroundColor(Color(isDisclosure ? tokens.disclosureIconForegroundColor : tokens.trailingItemForegroundColor))
+                                    .frame(width: isDisclosure ? disclosureSize : iconSize,
+                                           height: isDisclosure ? disclosureSize : iconSize)
+                                    .padding(.leading, isDisclosure ? tokens.disclosureInterspace : tokens.iconInterspace)
+                            }
+                        }
+                    }
+                }
+            })
+            .buttonStyle(ListCellButton(tokens: tokens, layoutType: layoutType))
+        }
+    }
+
+    struct ListCellButton: ButtonStyle {
+        let tokens: MSFListTokens
+        let layoutType: MSFListCellVnextLayoutType
+        func makeBody(configuration: Self.Configuration) -> some View {
+            return configuration.label
+                .contentShape(Rectangle())
+                .frame(minHeight: layoutType.height)
+                .listRowInsets(EdgeInsets())
+                .padding(.leading, tokens.horizontalCellPadding)
+                .padding(.trailing, tokens.horizontalCellPadding)
+                .background(configuration.isPressed ? Color(tokens.highlightedBackgroundColor) : Color(tokens.backgroundColor))
+        }
+    }
+
+    /// View for List Header (availble for iOS 14.0+)
+    struct Header: View {
+        let title: String
+        var tokens: MSFListTokens
+
+        init(title: String, tokens: MSFListTokens) {
+            self.title = title
+            self.tokens = tokens
+        }
+
+        var body: some View {
+            GeometryReader { _ in
+                HStack(spacing: 0) {
+                    Text(title)
+                        .font(Font(tokens.subtitleFont))
+                        .foregroundColor(Color(tokens.subtitleColor))
+                    Spacer()
+                }
+            }
+        }
+    }
+}
+
+@objc(MSFListVnext)
+open class MSFListVnext: NSObject {
+
+    private var hostingController: UIHostingController<MSFListView>
+
+    @objc open var view: UIView {
+        return hostingController.view
+    }
+
+    @objc open var state: MSFListVnextState {
+        return hostingController.rootView.state
+    }
+
+    @objc open var sections: [MSFListVnextSectionData] {
```

Comment:
Is there a specific reason why this is not part of the state object?
I think it makes sense to have the sections as part of the state so we get the list updated when rows/sections get added or removed.

Reply:
Updated to include inside state